go/encoding/basex/encoding.go

```diff
@@ -14,13 +14,14 @@ import (
 // An Encoding is a radix X encoding/decoding scheme, defined by X-length
 // character alphabet.
 type Encoding struct {
-	encode      []byte
-	decodeMap   [256]byte
-	inBlockLen  int
-	outBlockLen int
-	base        int
-	logOfBase   float64
-	baseBig     *big.Int
+	encode                 []byte
+	decodeMap              [256]byte
+	inBlockLen             int
+	outBlockLen            int
```

Comment:
It's a little annoying that when referring to Unicode strings, raw bytes are the "encoded" for, but when referring to BaseX, raw bytes are the "decoded" form :p